src/views/Miscellaneous.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-menu-button slot="start" />
+        <ion-title>{{ $t("Miscellaneous") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <main>
+        <section>
+          <ion-list>
+            <ion-list-header>{{ $t("Miscellaneous jobs") }}</ion-list-header>
+            <ion-item v-for="job in miscellaneousJobs" :key="job.jobId" @click="viewJobConfiguration(job)">
+              <ion-label>{{ job.jobName }}</ion-label>
+              <ion-badge v-if="job.runTime" color="light">{{ timeFromNow(job.runTime)}}</ion-badge>
```

Comment:
I know you know that this is not in the designs. So is this your personal design choice?

Reply:
No sir, by mistake I used ion-badge instead of ion-note as mentioned in Figma. Corrected it.